<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RundownGroupWidget</class>
 <widget class="QWidget" name="RundownGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>36</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameItem">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <widget class="QLabel" name="labelThumbnail">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>2</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../Resource.qrc">:/Graphics/Images/Group.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelLabel">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>2</y>
        <width>531</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelColor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelGroupColor">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>34</y>
        <width>54</width>
        <height>2</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelGpiConnected">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../Resource.qrc">:/Graphics/Images/GpiConnected.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelActiveColor">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>addToCombo()</slot>
 </slots>
</ui>
